zdx/mem.go

```diff
@@ -77,7 +77,8 @@ func (t *MemTable) open() {
 	if t.valType != nil {
 		cols = append(cols, zng.Column{"value", t.valType})
 	}
-	t.builder = zng.NewBuilder(t.zctx.LookupTypeRecord(cols))
+	typ, _ := t.zctx.LookupTypeRecord(cols)
```

Comment:
I've seen functions that have a "Must" variant for cases that are (allegedly) certain not to fail, and that panic instead of returning an error. (For example https://godoc.org/regexp#Compile vs https://godoc.org/regexp#MustCompile). 

Maybe we should use this here? Curious to hear what others think too.

Reply:
I followed this suggestion and I like it.
@nwt any opinion on this?

Reply:
I think it's an excellent idea!